### DISCUSSION TITLE
Use pgp Authentication key for cloning repo in passforios

### DISCUSSION THREAD

**Discussion by AmrithVengalath** [mssun/passforios#599] [Ideas]
Hi, it will be cool if we can use the Authentication key with the pgp key as ssh key for cloning repo like we do in android using [OpenKey-Chain](https://play.google.com/store/apps/details?id=org.sufficientlysecure.keychain&hl=en_IN&gl=US) and [Password Store](https://play.google.com/store/apps/details?id=dev.msfjarvis.aps&hl=en_IN&gl=US). I tried adding the secret-sub key in ASCII-armored format but entering the password returns an error. I can't clone using GitHub password option either since I have 2FA turned on.

![IMG_2777](https://user-images.githubusercontent.com/34616860/214909094-affce69f-ef20-4759-a25f-e2c6b0a48c42.jpg)

**Comment by SimplyDanny** [comment]
What about using an SSH key?

**Reply by AmrithVengalath** [reply]
Yeah it works with a different ssh key. I was suggesting that option since I like how it works on Password Store in android.

**Comment by AmrithVengalath** [comment]
Yeah it works with a different ssh key. I was suggesting that option since I like how it works on Password Store in android.



------- Original Message -------
